js/ui/map.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from 'react';
+import { StyleSheet, View, Image } from 'react-native';
+import { getHost, getPublishableKey } from '../helpers';
+
+let MapLibreGL;
+try {
+  MapLibreGL = require('@maplibre/maplibre-react-native');
```

Comment:
I'm surprised it lets you do this using ES modules 🤔

Reply:
Ah, everything is compiled to CommonJS: https://stackoverflow.com/a/46336345/631816